kotlin-native/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExportTranslatorMobile.kt

```diff
@@ -5,14 +5,15 @@
 
 package org.jetbrains.kotlin.backend.konan.objcexport
 
+import org.jetbrains.kotlin.backend.konan.UnitSuspendFunctionExport
 import org.jetbrains.kotlin.descriptors.ClassDescriptor
 import org.jetbrains.kotlin.descriptors.FunctionDescriptor
 
 class ObjCExportTranslatorMobile internal constructor(private val delegate: ObjCExportTranslatorImpl) : ObjCExportTranslator by delegate {
     companion object {
         fun create(namer: ObjCExportNamer): ObjCExportTranslatorMobile {
             val mapper = ObjCExportMapper(local = true)
-            return ObjCExportTranslatorMobile(ObjCExportTranslatorImpl(null, mapper, namer, ObjCExportProblemCollector.SILENT, false))
+            return ObjCExportTranslatorMobile(ObjCExportTranslatorImpl(null, mapper, namer, ObjCExportProblemCollector.SILENT, false, UnitSuspendFunctionExport.LEGACY))
```

Comment:
I'm not sure about this, on the other hand I couldn't find any instance of `ObjCExportTranslatorMobile.create` 🤔 in the codebase.

Reply:
cc @fkistner.

Reply:
Unfortunately hard coding legacy mode won't work for us. How about adding a `configuration: ObjCExportLazy.Configuration` parameter to this `create` method and using that?

Reply:
No, looks good! Seems to be an oversight on our end. Thanks!